README.md

```diff
@@ -215,6 +215,59 @@ This function returns a list of dataframes. Each dataframe is the standings for
 5      Cincinnati Reds   68  94  .420  35.5
 ```
 
+# Caching
+
+To facilitate faster data retrieval for repeated calls, caching may be used to save a local copy of the requested data.
+By default caching is disabled so as to respect a user's potential desire to not have their hard drive space used
+without their permission. However, enabling caching is simple.
+
+Caching can be turned on by including the caching module in the pybaseball.datahelpers submodule, and enabling the
+caching option like so:
+
+```python
+from pybaseball.datahelpers import caching
+
+caching.cache_config.enable()
```

Comment:
I would prefer
```python3
import pybaseball
pybaseball.cache.enable()
```
and correspondingly
```python3
pybaseball.cache.disable()
```

Reply:
Done. As much as I like having things categorized (e.g., datahelpers.caching), this does look cleaner.